tests/cbmc/proofs/s2n_stuffer_printf/s2n_stuffer_printf_harness.c

```diff
@@ -21,6 +21,18 @@
 #include "stuffer/s2n_stuffer.h"
 #include "utils/s2n_mem.h"
 
+int nondet_int(void);
+
+int vsnprintf(char *str, size_t size, const char *fmt, va_list ap)
+{
+    if(size > 0)
+        __CPROVER_havoc_slice(str, size);
+    (void)*fmt;
+    if(__CPROVER_OBJECT_SIZE(ap) > 0)
+        (void)*(char **)ap;
+    return nondet_int();
```

Comment:
I'd forgotten that we don't run clang-format on these :/ Can you run `clang-format -i tests/cbmc/proofs/s2n_stuffer_printf/s2n_stuffer_printf_harness.c` anyway, just to clean this one up? These if statements just don't match any of the rest of our codebase 😅

Reply:
Done!